examples/inference/README.md

```diff
@@ -0,0 +1,175 @@
+# PiPPY distributed inference for large models
+
+PiPPY helps to run very large models for inference by splitting the model into mutliple stages running on multiple GPUs.
+PiPPY make this easier by providing a auto split API that automates this process for user. 
+
+## How it works
+
+PiPPY splits your model into multiple stages, each stage loaded on one gpu then the input batch will be furhter divided into micro-batches and run through the splits from 
+rank0..rankN. Results are being returned to rank0 as its runing the PipelineDriver. Please read more on pipleines [here](https://github.com/pytorch/tau/blob/main/README.md)
+
+The flowchart below helps to visualize the process in high level as well.
+
+<img src="https://user-images.githubusercontent.com/9162336/206815839-60d43f93-ff4b-4a5e-99db-1e0c4a55a4ac.png" alt="drawing" width="400"/>
+
+
+## PiPPY support arbitary checkpoint splitting 
+
+Unlike most of the available solutions that they need to know the model architecture beforehand, PiPPY supports arbitary PyTorch checkpoints.
+* PiPPY supports both manual splitting and auto split.
+* Auto split uses `split_policy` and support both `equal_size` and `threshod` policies, the name are self-explanatory.
+* PiPPY use FX to trace and split the model.
+
+## Setting you need to care about
+
+* pp_group_size configure the number of pipeline parallelism group, meaning essentially on how many gpus our model to need be splitted and form a pipeline.
```

Comment:
the number of pipeline parallelism group
-->
the size of pipeline parallelism group